src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -44,6 +68,69 @@ public HelpWindow() {
         this(new Stage());
     }
 
+    /**
+     * Populates the table with all the command information.
+     */
+    private void setUpTable(TableView<CommandHelper> table, ObservableList<CommandHelper> commands) {
+        TableColumn<CommandHelper, String> commandTitle = new TableColumn<>("Command Title");
+        commandTitle.setCellValueFactory(new PropertyValueFactory<>("commandTitle"));
+
+        TableColumn<CommandHelper, String> commandUsage = new TableColumn<>("Command Usage");
+        commandUsage.setCellValueFactory(new PropertyValueFactory<>("commandUsage"));
+
+        table.setItems(commands);
+
+        table.setSelectionModel(null);
+        table.prefHeightProperty()
+                .bind(Bindings.size(table.getItems()).multiply(TABLE_ITEM_HEIGHT).add(ADDITIONAL_MARGIN));
+
+        commandTitle.prefWidthProperty().bind(table.widthProperty().multiply(0.20));
+        commandUsage.prefWidthProperty().bind(table.widthProperty().multiply(0.70));
+
+        table.getColumns().add(commandTitle);
+        table.getColumns().add(commandUsage);
+    }
+
+    private static ObservableList<CommandHelper> getGeneralCommands() {
+        return FXCollections.observableArrayList(
+                new CommandHelper("Open help panel", "help"),
+                new CommandHelper("List all", "list"),
+                new CommandHelper("Clear all data", "clear"),
+                new CommandHelper("Exit", "exit")
+        );
+    }
+
+    private static ObservableList<CommandHelper> getStudentCommands() {
+        return FXCollections.observableArrayList(
+                new CommandHelper("Add student", "add_student n/NAME p/STUDENT_PHONE_NUMBER "
+                        + "e/EMAIL a/ADDRESS l/STUDY_LEVEL g/GUARDIAN_PHONE_NUMBER r/RELATIONSHIP_WITH_GUARDIAN"),
+                new CommandHelper("Find student", "find_student KEYWORD [MORE_KEYWORDS]"),
+                new CommandHelper("Edit student", "edit_student STUDENT_INDEX [n/NAME]"
+                        + "[p/STUDENT_PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [l/STUDY_LEVEL] [g/GUARDIAN_PHONE_NUMBER]"
+                        + " [r/RELATIONSHIP_WITH_GUARDIAN]"),
```

Comment:
1. Missing a gap between `[n/NAME]` and `[p/STUDENT_PHONE_NUMBER]`
2. The sentence for `edit_student` seem to truncate on my side, i think there's a way to wrap text (?), refer to screenshot below!
![image](https://user-images.githubusercontent.com/45890870/113272502-43211980-930e-11eb-9c63-31af9e00857b.png)